docs/process.md

```diff
@@ -118,3 +119,82 @@ that is:
+
+If you package Emscripten for users in some manner, the above docs should be
+helpful with understanding versioning and so forth. This section goes into that
+in more detail.
+
+The core
+[DEPS](https://chromium.googlesource.com/emscripten-releases/+/refs/heads/master/DEPS)
+file in the chromium `emscripten-releases` contains all the information about versions
+in all the repos. This is the repo used by our CI to build and test emscripten. How
+this works is that you look at the DEPS file at a particular time (a git commit; that
+is a normal git repo). It has lines like these:
+```
+  'binaryen_revision': '06698d7a32cb4eeb24fea942e83d1b15e86a73e6',
+  'emscripten_revision': '7224b7930ec2a6abca332300e247619e1aea1719',
+  'llvm_project_revision': '33ef687d94604aeb73bedbcf3050524465a3439f',
+```
+Those are the git hashes in those repos. For building that git commit in
+emscripten-releases, our CI fetched and built those revisions.
+
+In principle you can build any git hash in that repo. You can look at the
+[CI UI](https://ci.chromium.org/p/emscripten-releases/g/main/console) to see
+if our CI shows green for any hash, and if so, it should be safe to build.
+
+Alternatively, you may want to build our official emscripten release tags, which
+are the versions the emsdk lets users install. To find the mapping between the
+emscripten versions and the git hash for the DEPS file, the emsdk has
+[emscripten-releases-tags.txt](https://github.com/emscripten-core/emsdk/blob/master/emscripten-releases-tags.txt).
+All versions listed there should be safe to build, as we check that the CI
+was green on them.
+
+To see how our CI builds things, the relevant script is
+[build.py](https://github.com/WebAssembly/waterfall/blob/master/src/build.py).
+In general, the repos you need to build are LLVM and Binaryen (as emscripten
+itself doesn't have any binaries to build). (Note: If you are packaging the older
+fastcomp compiler backend then you must also get the fastcomp and fastcomp-clang
+repos, and build them together to get LLVM and clang.)
+
+When packaging build results, you need the following executables (when using
+the wasm backend):
+
+  * From LLVM:
+    * clang
+    * clang++
+    * wasm-ld
+    * llc
+    * llvm-nm
+    * llvm-ar
+    * llvm-as
+    * llvm-dis
+    * llvm-dwarfdump
+  * From Binaryen:
+    * wasm-emscripten-finalize
+    * wasm-opt
+    * wasm-dis
+    * wasm-as
+    * wasm2js
+    * wasm-metadce
+
+(If you are packaging the older fastcomp backend instead of the wasm backend,
+you don't need wasm-ld or wasm2js, and you do need llvm-link and opt.)
+
+You also need to set up the `~/.emscripten` file for your users. Emscripten
+will try to do so on first run if such a file does not exist; the simplest
+thing is to look at those contents, edit the paths as needed if anything is
+wrong, and then use that file. (You can also look at how the emsdk generates
+the `.emscripten` file, which it does at the `activate step.) Some of the
```

Comment:
is there another page/link that describes the .emscripten file?

Reply:
I don't think so, the info is scattered around the main site I believe. This might be worth improving.